README.md

```diff
@@ -1 +1,35 @@
 # alphagov/tech-ops
+
+## Building and Releasing GovSVC Docker containers.
+
+The process for building and releasing the GovSVC specific Docker images is done by a set of Github actions attached to this repo.
+
+They will run automatically if a change is made to the corresponding dockerfile, or the can be triggered manually through the Github Actions UI.
+
+For running theses pipelines the run will have to be approved by a trusted person, that is in either the `re-autom8` or `reliability-engineering` Github teams.
```

Comment:
Perhaps better wording: "Pipeline runs will have to be approved by a trusted person ..."